app/src/main/java/com/yacgroup/yacguide/database/Rock.kt

```diff
@@ -21,23 +21,20 @@ import androidx.room.Entity
 import androidx.room.PrimaryKey
 
 @Entity
-class Rock {
-
-    @PrimaryKey
-    var id: Int = 0
-
-    var nr: Float = 0.toFloat()
-    var type: Char = ' '   // 'G' = Rock, 'M' = Massif, 'H' = Cage, 'B' = Boulder, 'N' = unofficial
-    var status: Char = ' ' // 'X' = prohibited, 'Z' = temporarily prohibited, 'T' = partly prohibited
```

Comment:
Why did you remove these comment?

Reply:
It's obvious since the constants are defined below.